sessiontxn/isolation/readcommitted.go

```diff
@@ -237,6 +242,9 @@ func (p *PessimisticRCTxnContextProvider) handleAfterPessimisticLockError(lockEr
 	}
```

Comment:
Should we call the `CancelAggressiveLocking` for the last else branch?
```
if deadlock, ok := errors.Cause(lockErr).(*tikverr.ErrDeadlock); ok && deadlock.IsRetryable {
} else if terror.ErrorEqual(kv.ErrWriteConflict, lockErr) {
} else {
  // This branch.
}
```

Reply:
The final `else` branch will not set `retryable = true` thus the function will finally return error. Cancelling here looks not necessary.